src/main/java/com/trivialTrivia/entity/Category.java

```diff
@@ -0,0 +1,19 @@
+package com.trivialTrivia.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
```

Comment:
Small change, but we could update this to import all from lombok and reduce 4 lines down to 1

Reply:
hey i added the asterisk down to one import line